librsvg.yaml

```diff
@@ -1,22 +1,21 @@
 #nolint:git-checkout-must-use-github-updates
 package:
   name: librsvg
-  version: 2.58.0
-  epoch: 1
+  version: 2.58.91
+  epoch: 0
```

Comment:
@sergio-chainguard  As we are trying to ignore this dev updates. We should revert all of these changes. and get back to version 2.58.0